packages/@okta/vuepress-site/docs/guides/pwd-optional-sign-in-email/main/nodeexpress/integrationsteps.md

```diff
@@ -0,0 +1,61 @@
+### 1: Create and display sign-in page
+
+Create a sign-in page that captures the user's username.
+
+<div class="common-image-format bordered-image">
+
+![Screenshot showing a sign-in page with the username field.](/img/pwd-optional/pwd-optional-sign-in-page.png)
+
+</div>
+
+### 2: Submit username
+
+When the user initiates the sign-in, pass the username to `OktaAuth.idx.authenticate()`.
+
+```javascript
+  const { username } = req.body;
+  const authClient = getAuthClient(req);
+  const transaction = await authClient.idx.authenticate({ username });
+```
+
+### 3. Verify identity with the email authenticator
+
+`OktaAuth.idx.authenticate()` returns an `IdxTransaction` object indicating that the user needs to verify their identity with the email authenticator challenge.
+
+```json
+{
+  status: "PENDING",
+  nextStep: {
+    name: "challenge-authenticator",
+    type: "email",
+    authenticator: {
+      type: "email",
+      key: "okta_email",
+      displayName: "Email",
+    },
+  },
+}
+```
+
+The email authenticator supports OTP and magic links, and you can integrate both methods into your application. Learn more about integrating the email authenticator challenge by visiting the [Okta email integration guide](/docs/guides/authenticators-okta-email/nodeexpress/main/#integrate-email-challenge-with-magic-links).
+
+### 4. Complete sign-up
```

Comment:
I know th epoint of the article is sign in with email only, but what about the optional authenticators the user has enrolled . Shouldn't there be a step, or at least an acknowledgement in the text, where optional authenticators are listed but skipped, as they are in the Sign-up guide.
```suggestion
### 4. Complete sign-in flow
```

Reply:
updated.

Reply:
Add a note since it's out of scope note of use case: In a MFA use case, Find the status, you would have to choose another authenticator and roll through the loop again. And a link to another guide.

Reply:
updated.